code/datums/components/chasm.dm

```diff
@@ -208,6 +210,10 @@ GLOBAL_LIST_INIT(chasm_storage, list())
 	SIGNAL_HANDLER
 	UnregisterSignal(gone, COMSIG_LIVING_REVIVE)
 
+///A type of chasm that bypasses the check to drop objects down a z-level, throwing whatever falls straight into the chasm contents.
+/datum/component/chasm/offstage
+	respect_zlevel = FALSE
```

Comment:
do you think this would benefit if we made this an argument on initialize instead of a subtype or

Reply:
yeah you already override addcomponent

Reply:
I would just make this an arg, default to TRUE to make admin's lives easier